### DISCUSSION TITLE
org.gnome.Platform.Compat.i386 extension

### DISCUSSION THREAD

**Discussion by mirkobrombin** [elementary/flatpak-platform#61] [Q&A]
I hope I'm not posting in the wrong place.

I'm trying to package Bottles for the elementary AppCenter but it make use of [org.gnome.Platform.Compat.i386](https://github.com/bottlesdevs/Bottles/blob/177beca6ece040a8e13a6551de035ae799f9edee/com.usebottles.bottles.json#L26)  extension which is not available from the elementary repository.

We need that to be able to compile some libraries and be able to run 32bit binaries via wine. 

I am relatively new to the world of Flatpak, let me know if I'm wrong please 😁

**Comment by mirkobrombin** [comment]
Nevermind, I was able to make it works manually bundling the dependencies from the AppImage
https://github.com/bottlesdevs/Bottles/blob/master/com.usebottles.bottles.elementary.yml

![image](https://user-images.githubusercontent.com/9479903/129542227-31c82ecc-d20b-482a-9974-f7cdc0af2083.png)

**Comment by alatiera** [comment]
I think this is a valid issue and something that should be mirrored into the repo.

**Reply by mirkobrombin** [reply]
> I think this is a valid issue and something that should be mirrored into the repo.

Can be useful to other applications 🤔
